pallets/amm/src/tests.rs

```diff
@@ -283,6 +283,149 @@ fn remove_zero_liquidity_should_not_work() {
 	});
 }
 
+#[test]
+fn remove_liquidity_should_not_work() {
```

Comment:
remove liquidity should work;-)

it should not work if requirements are not met.